### DISCUSSION TITLE
My Chia asset lost!

### DISCUSSION THREAD

**Discussion by mohamin1** [Chia-Network/chia-blockchain#8411] [Support]
hi there
I sent amount 1 XCH asset from kucoin exchang to Chia wallet (Receive Address) but I can't see that in Chia wallet!
please help me...
thanks.

**Comment by loppefaaret** [comment]
make sure the full node and the wallet are both fully synced

**Reply by mohamin1** [reply]
Thank you

**Reply by einoaarnioo** [reply]
> make sure the full node and the wallet are both fully synced

how to make fully synced both of them ?

**Reply by loppefaaret** [reply]
look under the "Full Node" tab, and it should display in green "Synced" in the "Status" pane
and under "Wallet" tab, in the top right, the "Wallet Status:" should also be in green, and only at most a few blocks behind the Full Node

**Reply by einoaarnioo** [reply]
> look under the "Full Node" tab, and it should display in green "Synced" in the "Status" pane and under "Wallet" tab, in the top right, the "Wallet Status:" should also be in green, and only at most a few blocks behind the Full Node

The status in the Full node section is: " Syncing 44,416/1,097,271" Wallet status is also "Not Synced (44,416)"  what should I do?

**Reply by loppefaaret** [reply]
how long have you been stuck on block 44,416 ? if it is written in yellow, and the number is increasing, it is syncing, and you just wait - if it has been stuck there for hours, you might have other issues

**Reply by einoaarnioo** [reply]
It is written in yellow but stuck there for hours; what other issues may exist?

**Reply by loppefaaret** [reply]
just following up after the weekend - if you are still stuck, i wanted to ask if you are able to locate your `debug.log` under your users folder `~\.chia\mainnet\log\` and attach it here
